securedrop_client/gui/actions.py

```diff
@@ -259,10 +259,11 @@ def _on_triggered(self) -> None:
         (Re-)generates the conversation transcript and opens a confirmation dialog to export it,
         in the manner of the existing ExportFileDialog.
         """
+        transcript_filename = _("{}_transcript.txt").format(self._source.journalist_filename)
```

Comment:
:thought_balloon: I should probably have extracted that constant earlier! :+1: 

If you feel like opening a tiny PR to do that refactoring (with `transcript.txt`), I'd review and approve that swiftly @philmcmahon:

Change:

```python
        transcript_filename = "transcript.txt"
```

Reply:
Also, the change maybe intentional, but note that the `transcript.txt` name isn't currently internationalized. :warning:

Reply:
Re internationalisation, thanks that's helpful to know for future PRs to the main branch!